monero-wallet-gui.pro

```diff
@@ -327,7 +327,38 @@ macx {
 
 
 # translation stuff
-TRANSLATIONS = $$files($$PWD/translations/monero-core_*.ts)
```

Comment:
Please remove all following strings and leave only `-TRANSLATIONS = $$files($$PWD/translations/monero-core_*.ts)`